CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 * Remove redundant `currency` parameter from `Unzer::chargePayment()` method.
 * Add "geoLocation" property to all payment type classes.
 * Several minor improvements.
+* Add account information coming from PAPI to Authorize class.
```

Comment:
Should it be: to Authorize/Charge classes ?

Reply:
No, only authorize, since the Charge class already had those properties.

Reply:
Ok I see, they were just extracted in the new class